Gruntfile.js

```diff
@@ -292,6 +253,14 @@ var _              = require('lodash'),
                     }
                 },
 
+                jshintClient: {
+                    command: 'cd core/client && grunt jshint'
+                },
+
+                jscsClient: {
+                    command: 'cd core/client && grunt jscs'
```

Comment:
This is better handled with `subgrunt` (see #6909) as it will ensure that all the necessary client packages are installed - the client has additional `jscs` rules pulled in via the `ember-suave` npm package.